src-tauri/src/commands/system_tray.rs

```diff
@@ -0,0 +1,83 @@
+use tauri::Manager;
+use tauri::{
+    AppHandle, CustomMenuItem, Runtime, SystemTray, SystemTrayEvent, SystemTrayMenu,
+};
+
+use crate::SYSTEMTRAY_MANAGER;
+
+const QUIT: &'static str = "quit";
+
+#[derive(Debug, Clone)]
+pub struct SystemTrayManager {
+    pub exit_in_tray: bool,
+    pub start_in_tray: bool,
+}
+
+impl SystemTrayManager {
+    pub fn new() -> SystemTrayManager {
+        SystemTrayManager { exit_in_tray: false, start_in_tray: false, }
+    }
+    
+    // pub fn handle_window_exit<R: Runtime>(&self) -> impl Fn(GlobalWindowEvent<R>) + Send + Sync + 'static {
+    //     return |event| match event.event() {
+    //         tauri::WindowEvent::CloseRequested { api, .. } => {
+    //             event.window().hide().unwrap();
+    //             api.prevent_close();
+    //         }
+    //         _ => {}
+    //     }
+    // }
+}
+
+// Initializes the system tray with menus.
+pub fn init_system_tray() -> SystemTray {
+    // Menus
+    let menu_quit = CustomMenuItem::new(QUIT, "Quit");
```

Comment:
We should probably find a way of localizing this as well, but I understand there's currently not yet a nice way of getting the translations to the rust side. I think doing that might be a bit outside of this PR's scope, especially when you consider the translations should change when the user switches language on the front, so that's probably something to look into after getting this merged.

tl;dr: this is fine as is, feel free to ignore. I've opened a separate issue for this in #23.